src/App.vue

```diff
@@ -4,11 +4,9 @@
       <router-view v-if="customLayout"/>
       <v-container v-else class="full-height-content">
         <v-row >
-          <v-col cols="1"></v-col>
-          <v-col cols="10">
+          <v-col>
```

Comment:
Idea: if we now have a row that contains a single column, there is no need for a grid system anymore, and we could also just remove both the `v-row` and the `v-col` entirely with the `v-container` making sure everything stays in place.

Reply:
I am leaving container in this case because of the class.